### ISSUE TITLE
1 class object detection setup

### ISSUE THREAD

**Issue by planemanner** [zzzxxxttt/pytorch_simple_CenterNet_47#7]
Hi, thanks for your great job.

I have a question about training for 1-class object detection setup. (custom dataset)

I already convert own dataset to coco json format and split 3 parts (train, validation, test)

and revised serveral variables and argument in ./datasets/coco.py 

COCO_NAMES =  ['__background, target_object']
COCO_IDS = [1]
self.num_classes = 1
 
after that, I tuned the learning rate for and batch size and trained.

But I've found it was totally not trained through evaluation step. 

What should I configure more in your repository?

Can you suggest something?

**Comment by zzzxxxttt** [comment]
There are actually 2 classes (background & target) in your case, so try ```COCO_IDS = [0, 1]``` and ```self.num_classes = 2```
